lib/views/chatroom/views/chatroom_participants_page.dart

```diff
@@ -79,67 +82,55 @@ class ParticipantItem extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      width: getWidth(context),
-      height: 72,
-      padding: const EdgeInsets.symmetric(horizontal: 24),
-      child: Row(
-        children: [
-          Container(
-            width: 40,
-            height: 40,
-            decoration: BoxDecoration(
-              shape: BoxShape.circle,
-              image: DecorationImage(
-                image: NetworkImage(profilePic),
-                fit: BoxFit.cover,
-              ),
+    return GestureDetector(
+      onTap: () {
+        Route route = MaterialPageRoute(
+          builder: (BuildContext context) => BlocProvider<ProfileBloc>(
+            create: (BuildContext context) =>
+                ProfileBloc()..add(InitProfileEvent()),
+            child: const ProfilePage(
+              isSelf: false,
             ),
           ),
-          const SizedBox(width: 16),
-          Text(
-            name,
-            style: GoogleFonts.montserrat(
-              fontSize: 16,
-              fontWeight: FontWeight.w600,
+        );
+        Navigator.push(context, route);
```

Comment:
I guess named routing is a better option.
While navigation without using named routes is OK for smaller projects, in more complex apps it adds code duplication.

Reply:
I will be implementing all the routes using the `go_router` package as soon as we start making this more and more complex, the job right now is to nail the UI part the best we can. I get your point, but it is anyway going to be added.

Reply:
Okay, but I guess there's no need to implement routing using a package, the primary navigator has enough options to satisfy app navigation.